src/modules/ipfix/aggregator/AggregatorBaseCfg.cpp

```diff
@@ -178,6 +182,9 @@ Rule::Field* AggregatorBaseCfg::readFlowKeyRule(XMLElement* e) {
 		ruleField->type.enterprise = ie.getEnterpriseNumber();
 		ruleField->type.length = ie.getIeLength();
 
+		if (BaseHashtable::isToBeAggregated(ruleField->type))
```

Comment:
Can you add braces around the then-part of the `if` statement?

Reply:
I'm used to kernel style. :-) Added the braces.